### ISSUE TITLE
Windows mention is now a tad outdated

### ISSUE THREAD

**Issue by tianon** [antirez/redis-io#123]
Currently, the download page mentions the following:

> The Redis project does not officially support Windows. However, the Microsoft Open Tech group develops and maintains this Windows port targeting Win64.

Given https://github.com/MicrosoftArchive/redis/issues/525 and that https://github.com/MSOpenTech/redis now redirects to https://github.com/MicrosoftArchive/redis ("Archived Microsoft GitHub repositories - for historical interest, not actively maintained"), would it make sense to remove that mention from the download page? (or at least change it to make it clear Microsoft is no longer maintaining the fork?)

I'd be happy to try a PR if it'd be helpful to do so (with some guidance on the preferred direction to take this ala "remove the reference" or "mention the abandonment"). :+1:

**Comment by tporadowski** [comment]
@tianon: Yes, unfortunately that port to Windows is no longer maintained, so I've spent some time and I'm trying to update it to latest Redis 4.0.2. [This is my fork of MSOpenTech's port to Windows](https://github.com/tporadowski/redis) updated to 4.0.2 which is close to reaching its "alpha" milestone (all Redis tools updated and buildable, without modules support). I don't dare to have it mentioned on redis.io webpage, but maybe one day ;)
